integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/logsearch/LogSearchUtil.java

```diff
@@ -0,0 +1,61 @@
+package com.sequenceiq.it.cloudbreak.newway.logsearch;
+
+import static com.sequenceiq.it.cloudbreak.newway.CloudbreakTest.LOG_SEARCH_QUERY_TYPES;
+import static com.sequenceiq.it.cloudbreak.newway.CloudbreakTest.LOG_SEARCH_URL_PREFIX;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.assertj.core.util.Lists;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.sequenceiq.it.IntegrationTestContext;
+
+public class LogSearchUtil {
+
+    public static final String LOG_SEARCH_CBID_QUERY_TYPE = "cbid";
+
+    public static final Long LOG_SEARCH_CBID_ID = 1L;
+
+    public static final String LOG_SEARCH_CBNAME_QUERY_TYPE = "cbname";
+
+    public static final Long LOG_SEARCH_CBNAME_ID = 2L;
+
+    public static final String LOG_SEARCH_COMPONENT_QUERY_TYPE = "component";
+
+    public static final Long LOG_SEARCH_COMPONENT_ID = 3L;
+
+    private LogSearchUtil() {
+
+    }
+
+    public static void addQueryModelForLogSearchUrlToContext(IntegrationTestContext context, Long id, String type, String value) {
+        Optional<QueryType> queryType = ((List<QueryType>) context.getContextParam(LOG_SEARCH_QUERY_TYPES, List.class))
+                .stream()
+                .filter(logSearchQueryType -> org.apache.commons.lang3.StringUtils.equals(logSearchQueryType.getId(), type))
+                .findFirst();
+        if (queryType.isPresent()) {
```

Comment:
unnecessary if statement. the previous lambda could be extended by ifPresent() like this:

`((List<QueryType>) context.getContextParam(LOG_SEARCH_QUERY_TYPES, List.class))
                .stream()
                .filter(logSearchQueryType -> org.apache.commons.lang3.StringUtils.equals(logSearchQueryType.getId(), type))
                .findFirst().ifPresent(queryType1 -> context.putCleanUpParam(type, QueryModel.create(id, queryType1, value)));`